SonataPageBundle.php

```diff
@@ -59,10 +59,18 @@ public function boot()
         }
 
         call_user_func(array($class, 'setSlugifyMethod'), function ($text) use ($container) {
+            // NEXT_MAJOR: remove this check
```

Comment:
Not really true. The check is not deprecated. The default service fallback value is.